tests/unit/modules/tuned_test.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from salt.modules import tuned
+
+from salttesting import skipIf, TestCase
```

Comment:
Looks like pylint / lint is not happy with those import : https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/10010/violations/file/tests/unit/modules/tuned_test.py/

Maybe importing tests.support.XXX will fix it as it is done in another (random) I checked : https://github.com/saltstack/salt/blob/develop/tests/unit/modules/test_archive.py

Hope this can help :)